index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Maybe move this to `src/index.html` as well?

Reply:
done! thanks.